tempesta_fw/sock.c

```diff
@@ -1323,13 +1340,15 @@ ss_getpeername(struct sock *sk, TfwAddr *addr)
 }
 EXPORT_SYMBOL(ss_getpeername);
 
-#define __sk_close_locked(sk)					\
-do {								\
-	ss_do_close(sk);					\
-	bh_unlock_sock(sk);					\
-	SS_CALL_GUARD_EXIT(connection_drop, sk);		\
```

Comment:
What is the reason to drop call of  `SS_CALL_GUARD_EXIT`?

Reply:
Yes. Did not have time to push